docs/usage/faq.md

```diff
@@ -1,18 +1,24 @@
 # Frequently Asked Questions
 
-## How can I load external Python files in Pyodide?
+## How can I load external files in Pyodide?
 
-The two possible solutions are,
+The three possible solutions are,
 
-- include these files in a Python package, build a pure Python wheel with
+- include the files in a Python package, build a pure Python wheel with
   `python setup.py bdist_wheel` and
   {ref}`load it with micropip <micropip-installing-from-arbitrary-urls>`.
-- fetch the Python code as a string and evaluate it in Python,
+- fetch Python code as a string and evaluate it:
   ```js
   pyodide.runPython(await (await fetch("https://some_url/...")).text());
   ```
+- Use `pyodide.FS.writeFile(path, data)` to directly create a file inside the
+  file system. Note that the directory must already exist, if not use
+  `pyodide.FS.mkdir` first. The file system API is documented
```

Comment:
That exists, by why would we be recommending that as opposed to python `open`, `os.mkdir` etc? Unless one would like to do it on the JS side. 

Also it's not really an alternative solution to fetch above, I would say, it still would need to be fetched somehow.